lib/ui/discover/discover_detail_screen.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:farmsmart_flutter/data/model/article_entity.dart';
+import 'package:farmsmart_flutter/utils/assets.dart';
+import 'package:farmsmart_flutter/utils/colors.dart';
+import 'package:farmsmart_flutter/utils/dimens.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_html/flutter_html.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:farmsmart_flutter/redux/app/app_state.dart';
+import 'package:farmsmart_flutter/utils/styles.dart';
+
+import 'discover_viewmodel.dart';
+
+class ArticleDetailScreen extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() {
+    return _ArticleDetailState();
+  }
+}
+
+class _ArticleDetailState extends State<ArticleDetailScreen> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: StoreConnector<AppState, DiscoverViewModel>(
+          builder: (_, viewModel) =>
+              _buildBody(context, viewModel.selectedArticle),
+          converter: (store) => DiscoverViewModel.fromStore(store)),
+    );
+  }
+
+  Widget _buildBody(BuildContext context, ArticleEntity selectedArticleData) {
+    return Scaffold(
+          appBar: AppBar(
+            // Define custom appbar here
+            automaticallyImplyLeading: true,
+          ),
+          body : Container(
+              decoration: BoxDecoration(
+                color: Color(white),
+                borderRadius: BorderRadius.circular(8.0),
+              ),
+              child: ListView(children: <Widget>[
+                FadeInImage.assetNetwork(
+                    image: selectedArticleData.imageUrl,
+                    height: listImageHeight,
+                    width: listImageWidth,
+                    placeholder: Assets.IMAGE_PLACE_HOLDER,
+                    fit: BoxFit.fitWidth),
+                Padding(
+                    padding: Margins.boxBigPadding(),
+                    child: Column(
+                      crossAxisAlignment: CrossAxisAlignment.start,
+                      children: <Widget>[
+                        Text(selectedArticleData.title, style: Styles.titleTextStyle()),
+                        Margins.generalListMargin(),
+                        Html(data: selectedArticleData.content),
+                      ],
+                    )),
```

Comment:
Can we move the Widget to a method that can return it, and then another one to return the Padding?

We would have something like, I think is more readable:
```
  child: ListView(children: <Widget>[
                getFadeImage(selectedArticleData),
                getPadding(selectedArticleData),
              ]))
```

Reply:
Done !

Reply:
Albert, don't copy my comments! xD

Reply:
@emilianowearemobile I haven't seen yours! <image src="https://user-images.githubusercontent.com/15966505/57441666-b0bf8a80-724a-11e9-99f0-358d240eb456.png" width=8%/>